src/client/ns_turn_msg.c

```diff
@@ -567,7 +567,7 @@ int stun_is_challenge_response_str(const uint8_t *buf, size_t len, int *err_code
       const uint8_t *value = stun_attr_get_value(sar);
       if (value) {
         size_t vlen = (size_t)stun_attr_get_len(sar);
-        vlen = min(vlen, STUN_MAX_REALM_SIZE);
```

Comment:
C++ is a tad bit more picky about function arguments to `std::min`, so casting to size_t (to match `vlen`) fixes the compile error without any change in behavior.